lib/blacklight/configuration.rb

```diff
@@ -558,6 +565,22 @@ def add_show_tools_partial(name, opts = {})
       add_action(show.document_actions, name, opts)
       klass && ActionBuilder.new(klass, name, opts).build
     end
+
+    # Add a partial to the header when rendering a document.
+    # @!macro partial_if_unless
+    #   @param name [String] the name of the document partial
+    #   @param opts [Hash]
+    #   @option opts [Class] :component draw a component
+    #   @option opts [String] :partial partial to draw if component is false
+    #   @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true. The proc will receive the action configuration and the document or documents for the action.
+    #   @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true. The proc will receive the action configuration and the document or documents for the action.
+    def add_header_tools_partial(name, opts = {})
```

Comment:
I think we want to make this clear that this is the "show" header.  Not the header on the index view.

Reply:
how do you like add_show_header_tools_partial?